managed/provider.go

```diff
@@ -156,6 +156,7 @@ func (p *provider) GetResources(_ context.Context) (map[string]tfsdk.ResourceTyp
 		"ybm_metrics_exporter":                   resourceMetricsExporterType{},
 		"ybm_associate_metrics_exporter_cluster": resourceAssociateMetricsExporterClusterType{},
 		"ybm_associate_db_audit_export_config_cluster": resourceAssociateDbAuditExportConfigClusterType{},
+		"ybm_telemetry_provider":                       resourceTelemetryProviderType{},
```

Comment:
Seems your editor did not format the code.

Reply:
I ran the command `go fmt`. Is that not enough? The checks have passed

Reply:
I don't think we have a lint or format check

Reply:
Looks like we do in the `check formating` section here - https://github.com/yugabyte/terraform-provider-ybm/actions/runs/8263759565/job/22605881827?pr=103  This works?

Reply:
I thought was only for the docs. Then I guess seems ok :)